apps/prairielearn/src/sync/course-db.ts

```diff
@@ -1001,14 +1012,10 @@ function checkAllowAccessDates(rule: { startDate?: string; endDate?: string }):
       `Invalid allowAccess rule: startDate (${rule.startDate}) must not be after endDate (${rule.endDate})`,
     );
   }
-  let dateInFuture = false;
-  if (startDate && isFuture(startDate)) {
-    dateInFuture = true;
-  }
-  if (endDate && isFuture(endDate)) {
-    dateInFuture = true;
-  }
-  return { errors, dateInFuture };
+  return {
+    errors,
+    accessibleInFuture: !endDate || isFuture(endDate),
+  };
```

Comment:
@jonatanschroeder I changed this since your last review; this is now using the same (though opposite) check as is used when computing `courseInstanceExpired`. I'd appreciate eyes on this.